modules/oadp-backing-up-dpa-configuration-1-5-0.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-5-release-notes.adoc
+:_mod-docs-content-type: PROCEDURE
+
+[id="oadp-backing-up-dpa-configuration-1-5-0_{context}"]
+= Backing up the DPA configuration
```

Comment:
@apurvabhide17 sorry, I don't understand why this has been added to the `Upgrading OADP 1.4.0 to 1.5.0` section